airflow/www/views.py

```diff
@@ -1349,21 +1361,26 @@ class GraphForm(DateTimeWithNumRunsWithDagRunsForm):
         form = GraphForm(data=dt_nr_dr_data)
         form.execution_date.choices = dt_nr_dr_data['dr_choices']
 
-        task_instances = {
+        task_instances_dict = {
```

Comment:
not sure the coding style, but could we not put type in the variable name(https://docs.quantifiedcode.com/python-anti-patterns/readability/putting_type_information_in_a_variable_name.html) ?

Reply:
I know but did had a conflict because task_instances did already live there but did mean something else. I now did rename it to task_instances_json to be more clear